apps/errant-tower-client/src/common/styles/mixins.scss

```
@@ -59,3 +59,34 @@
         transparent
     );
 }
+
+@mixin primaryDividerLeft {
```

Stylelint (17.14.0)

[error] Expected mixin name "primaryDividerLeft" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)